src/assets.ts

```diff
@@ -272,6 +301,34 @@ export function createWebLaunchConfiguration(programPath: string, workingDirecto
 }`;
 }
 
+export function createBlazorWebAssemblyLaunchConfiguration(workingDirectory: string): string {
+    return `
+{
+    "name": "Launch Blazor web assembly project in Chrome",
+    "type": "pwa-chrome",
+    "request": "launch",
+    // If you have changed the default port / launch URL make sure to update the expectation below
+    "url": "https://localhost:5001",
```

Comment:
Leaving this as the https variant given I assume we want that as the default.

Reply:
Yes. Technically it would be possible to infer the HTTPS port from `launchSettings.json` but 5001 is indeed the default.

Inferring it doesn't make a huge difference because (1) the default is already correct, and (2) for a developer who's in the habit of changing it, once they've generated `launch.json` there would be nothing keeping it in sync with subsequent changes in `launchSettings.json`.

So basically I think this is fine!